doc/source/whats_new.rst

```diff
@@ -44,8 +44,8 @@ Changelog
    - Add methods ``get_channel_positions``, ``set_channel_positions`` for setting channel positions,
      and ``plot_projs_topomap`` to ``Raw``, ``Epochs`` and ``Evoked`` objects by `Teon Brooks`_
 
-   - Add EEG bad channel interpolation method (based on spherical splines) to ``Raw``, ``Epochs`` and ``Evoked`` objects
-     by `Denis Engemann`_
+   - Add EEG and MEG bad channel interpolation method (based on spherical splines) to ``Raw``, ``Epochs`` and ``Evoked`` objects
+     by `Denis Engemann`_ and `Mainak Jas`_
```

Comment:
We should somewhere write down some of our local style preferences in some
contrib file.

- return self
- single quotes
- generator / list comprehension instead of filter et al
- explicit functional constructors for builtin containers, dict() et al
- avoid nested functions if not necessary
- ...


2015-04-28 22:31 GMT+02:00 Eric Larson <notifications@github.com>:

> In doc/source/whats_new.rst
> <https://github.com/mne-tools/mne-python/pull/2011#discussion_r29285021>:
>
> > @@ -44,8 +44,8 @@ Changelog
> >     - Add methods ``get_channel_positions``, ``set_channel_positions`` for setting channel positions,
> >       and ``plot_projs_topomap`` to ``Raw``, ``Epochs`` and ``Evoked`` objects by `Teon Brooks`_
> >
> > -   - Add EEG bad channel interpolation method (based on spherical splines) to ``Raw``, ``Epochs`` and ``Evoked`` objects
> > -     by `Denis Engemann`_
> > +   - Add EEG and MEG bad channel interpolation method (based on spherical splines) to ``Raw``, ``Epochs`` and ``Evoked`` objects
> > +     by `Denis Engemann`_ and `Mainak Jas`_
>
> So maybe Add channel interpolation for EEG (spherical splines) and MEG
> (field interpolation) to ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2011/files#r29285021>.
>